ui/components/app/tab-bar/index.scss

```diff
@@ -53,6 +56,17 @@
           min-height: 14px;
         }
       }
+
+
+      &__icon {
+        margin-inline-end: 16px;
+      }
+
+      &__title {
+        @include Paragraph;
+
+        font-size: 1.25rem;
```

Comment:
[REQ]The figma file references 1.125rem for this item, or 18px. which is our @H4 typography mixin, not @Paragraph. My preference would be to drop this class altogether and use the <Typography> component instead as no major styles are required.

Reply:
Clear. Updated.